assets/services/certs/up

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/execlineb -P
+define PREFFIX "[oneshot] certs:"
 
+foreground {
+  if { s6-echo "${PREFFIX} ${C031}begin generating certs${C000}" }
```

Comment:
What is the reason to run this as `if` ?

Reply:
That's how it was done in postgres-prepare/up.

I removed it from this script, it still works.